rbc/tests/test_omnisci_caller.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+import pytest
+from rbc.externals.omniscidb import set_output_row_size
+from rbc.tests import omnisci_fixture, assert_equal
+
+
+@pytest.fixture(scope='module')
+def omnisci():
+    for o in omnisci_fixture(globals(), minimal_version=(5, 8)):
+        define(o)
+        yield o
+
+
+def define(omnisci):
+
+    @omnisci('T(T)', T=['int32', 'int64', 'float32', 'float64'])
+    def myincr(x):
+        return x + 1
+
+    T = ['int64', 'float64', 'int32']
+
+    @omnisci('UDTF(int32 size, T x0, OutputColumn<T> x)', T=T, devices=['cpu'])
+    def arange(size, x0, x):
+        set_output_row_size(size)
+        for i in range(size):
+            x[i] = x0 + x.dtype(i)
+        return size
+
+    T = ['int64', 'float32']
+
+    @omnisci('UDTF(Column<T> x, T dx, OutputColumn<T> y)', T=T, devices=['cpu'])
+    def aincr(x, dx, y):
+        size = len(x)
+        set_output_row_size(size)
+        for i in range(size):
+            y[i] = x[i] + dx
+        return size
+
+
+def test_remote_udf_evaluation(omnisci):
+    myincr = omnisci.get_caller('myincr')
```

Comment:
This mechanism is equivalent to an import in a Python world right?

Reply:
Not quite. The `get_caller` returns `CommonNameCallers` instance that may contain more than one Caller instance.

A user typically would not use `get_caller` method. In tests, we define UDF/UDTFs in a fixture and `get_caller` provides a way to retrieve the corresponding definitions from a test function.

Reply:
Oh I see, it gives back the undecorated function.